build/flags.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f "build/env.sh" ]; then
+    echo "$0 must be run from the root of the repository."
+    exit 2
+fi
+
+GO_ETHEREUM_FLAGS=""
```

Comment:
Note that the flags are modified in two places - how can they be tracked between these if not using a variable?